assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -136,14 +135,7 @@ function ExecutionResults({
           onChange={(newPredicates) => setPredicates(newPredicates)}
         />
       </div>
-      {cloudProvider === UNKNOWN_PROVIDER && (
-        <WarningBanner>
-          The following results are valid for on-premise bare metal platforms.
-          <br />
-          If you are running your HANA cluster on a different platform, please
-          use results with caution
-        </WarningBanner>
-      )}
+      {warningBanners[cloudProvider] ?? null}
```

Comment:
Just a headsup: changes in this file might conflict with what is happening in `new-checks-ui` branch.
We need to be careful when this PR gets merged and `new-checks-ui` gets rebased.

Reply:
Thank you for the heads-up @nelsonkopliku! I will keep an eye out for that PR.